src/main/java/com/odysseusinc/arachne/executionengine/model/descriptor/converter/DescriptorConverter.java

```diff
@@ -15,7 +17,8 @@ public RuntimeEnvironmentDescriptorDTO toDto(Descriptor model) {
         dto.setId(model.getId());
         dto.setBundleName(model.getBundleName());
         dto.setLabel(model.getLabel());
-        List<ExecutionRuntimeDTO> executionRuntimes = model.getExecutionRuntimes().stream()
+        List<ExecutionRuntime> modelExecutionRuntimes = Arrays.asList(model.getExecutionRuntimes());
```

Comment:
Arrays.stream can be used to simplify these two lines of code. It's just recommendation, not a mistake.